Sources/Target/POSIX/Thread.cpp

```diff
@@ -119,6 +119,8 @@ ErrorCode Thread::resume(int signal, Address const &address) {
                                    info, signal, address));
     _state = kRunning;
     _stopInfo.signal = 0;
+    _stopInfo.reason = StopInfo::kReasonNone;
```

Comment:
Then should we also remove `_stopInfo.signal = 0` seems inconsistent to only clear part of the state.